### DISCUSSION TITLE
sdk.all_running_queries & sdk.kill_query does not work as expected

### DISCUSSION THREAD

**Discussion by gokhulg** [looker-open-source/sdk-codegen#1351] [General]
Hi,

- The runtime parameter in the response value of all_running_queries is always 0.0.
- Passing a query-id to kill_query returns a **404-Looker Not Found** error. 

Goal: To automate killing of long-running queries. I can figure out runtime from created_at parameter, but the kill_query function itself does not work.
Please let me know if these issues can be fixed or if there is another reason to it.

**Comment by neil-richardson** [comment]
I'm getting the same issue.  Just started to get a process in place to kill long running queries and noticed the runtime is always 0.0.  Not found any way around this yet
